manifests/base/runtime-tensorflow-imagestream.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   tags:
     - annotations:
         # language=json
-        opendatahub.io/runtime-image-metadata: |
+        opendatahub.io/runtime-image-metadata: 
```

Comment:
huh? this works?

Reply:
json in yaml works, learned it today :)
https://alisoftware.github.io/yaml/2021/08/17/yaml-part1-json/#:~:text=cases%20of%20YAML.-,Back%20to%20Basics,-I%E2%80%99m%20sure%20most

Reply:
sure, but it changes the value of `opendatahub.io/runtime-image-metadata` from string to an array, that's ok?

Reply:
you are absolutely right
i thought was alright, seems like operator does have problem with patching.
reverted it back.